modules/oe_theme_helper/tests/src/Unit/TwigExtensionTest.php

```diff
@@ -300,4 +303,110 @@ public function toEclIconProvider(): array {
     ];
   }
 
+  /**
+   * Tests trimming of textfield.
+   *
+   * @param mixed $textfield
+   *   The string, object or render array.
+   * @param array $context
+   *   The context array.
+   * @param mixed $expected_result
+   *   The expected result.
+   *
+   * @covers ::trimTextfield
+   * @dataProvider trimTextfieldProvider
+   */
+  public function testTrimTextfield($textfield, array $context, $expected_result): void {
```

Comment:
No need to send the $context, you can send only the $length and then set it in the $context array.

Reply:
Nevermind..better send the context already prepared with the length.